cli/cli.go

```diff
@@ -256,7 +256,7 @@ func (cli *CLI) RunServer(ctx context.Context, args Args) error {
 		name := typeName[1]
 
 		switch typ {
-		case "musicbrainz", "ern", "eidr", "cwr":
+		case "musicbrainz", "ern", "eidr", "cwr", "identity", "claim":
```

Comment:
Aren't the `identity` and `claim` indexes backed by the same SQLite3 db?

Reply:
right.
so access claim and identity will be both via /meta-id/ path ?

Reply:
They are already aren't they? But yes, there should be one API for identity and claims by those identities, stored in the same SQLite3 db.